code/modules/clothing/spacesuits/rig.dm

```diff
@@ -66,6 +67,7 @@
 	desc = "An advanced suit that protects against hazardous, low pressure environments. Shines with a high polish."
 	item_state = "ce_hardsuit"
 	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECITON_TEMPERATURE
+	flags = FPRINT | TABLEPASS | HEADCOVERSEYES | BLOCKHAIR | STOPSPRESSUREDMAGE | PLASMAGUARD

```

Comment:
You copied and pasted them from helmet. Several of these flags, like BLOCKHAIR and HEADCOVERSEYES, are only intended for hats.
The rig suit inherits its flags from /obj/item/clothing/suit/space, which has 
`    flags = FPRINT  | STOPSPRESSUREDMAGE`
This is what should be before plasmaguard.